projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/MultiProgress.svelte

```diff
@@ -34,6 +34,8 @@
 					isLink={hasIssues}
 					class="jb-navigator-link"
 					to="/image-size-analysis/{group}/1"
+					trackEvent="clicked_isa_settings_group"
```

Comment:
What if we name this `clicked_isa_summary_group` instead? With the upcoming architecture changes to the UI, who knows if the summary would be on the main Settings page :D

Reply:
Changed it to "clicked_isa_group_on_summary_page" to make is similar to "clicked_view_isa_report_on_summary_page"